src/System.Diagnostics.Process/tests/System.Diagnostics.Process.Tests/System.Diagnostics.Process.Tests.csproj

```diff
@@ -18,10 +18,10 @@
   <ItemGroup>
     <Compile Include="Interop.cs" />
     <Compile Include="ProcessTest.cs" />
+    <Compile Include="$(CommonPath)\Interop\Interop.PlatformDetection.cs" />
```

Comment:
I was including and excluding files during testing from project, and VS has automatically changed the lines. Will undo this.